react/src/DealDetail.js

```diff
@@ -545,15 +545,16 @@ export function DealStatusInfo(props) {
                 </p>
             </p>
             <p>
-                <i>Announcing</i>
+                <i>Indexing</i>
                 <p>
-                    Boost is announcing the deal to the network so that clients know where to retrieve it.
+                    Boost is indexing the deal in Local Index Directory(LID)
+                    and will be announcing the deal to the network so that clients know where to retrieve it.
```

Comment:
```suggestion
                    Boost is indexing the deal in the Local Index Directory(LID)
                    and will be announcing the deal to the network so that clients know where to retrieve it from.
```